### DISCUSSION TITLE
Author bio as subtitle in profile layout.

### DISCUSSION THREAD

**Discussion by dsluo** [jpanther/congo#78] [Ideas]
E.g.:

![image](https://user-images.githubusercontent.com/5554733/148681686-c0806868-2e81-41ef-823c-45b41619acd9.png)

This would provide a built-in solution to some of the confusion in #38. I have a pretty quick implementation here: dsluo/congo@788f1a6e9a60b68d8a316780900f73bb1f54e980.

Alternatively, and perhaps preferably, there should be a separate `subtitle` option that defaults to the author bio, but I don't know if Hugo has an option for custom options in `config.toml`.

**Comment by jpanther** [comment]
I agree that the current way this works is confusing. I'll look at options to improve this in a future release.
